app/assets/javascripts/inventory.js

```diff
@@ -0,0 +1,39 @@
+var sendData = function(id) {
+	book_id = '.book_' + id.toString();
+	dataObjects = $(book_id).serializeArray();
+
+	$.ajax({
+		type: "POST",
+		url: '/update_checklist_item',
+		data: {
+			id: id,
+			book: {
+				title: dataObjects[0].value,
+				publisher: dataObjects[1].value,
+				publish_date: dataObjects[2].value,
+				publication_place: dataObjects[3].value,
+				language: dataObjects[4].value,
+				pages: dataObjects[5].value,
+				location: dataObjects[6].value,
+				count: dataObjects[7].value
+			}
+		},
+		dataType: 'json',
+		success: function(data) {
+			if(data.status === 'ok'){
+				displayMessage("Saved!");
```

Comment:
@forplayplay I like the idea of here also resetting the form values to match the updated book info that the controller would send back. So, if editing part of it fails, the user knows because they're looking at exactly what the data is.